pkg/apis/kops/componentconfig.go

```diff
@@ -992,6 +992,8 @@ type ClusterAutoscalerConfig struct {
 	// CPURequest of cluster autoscaler container.
 	// Default: 100m
 	CPURequest *resource.Quantity `json:"cpuRequest,omitempty"`
+	// MaxNodeProvisionTime determines how long CAS will wait for a node to join the cluster.
+	MaxNodeProvisionTime *string `json:"maxNodeProvisionTime,omitempty"`
```

Comment:
Why is this a pointer? What is the utility of setting this to the empty string?

Reply:
I suppose that is the case for many of the above. Empty string and nil would mean the same.

Do you want to change this to non-pointer if we prefer that?